From 06c5593fd110d235e10643d2c197d948d2ec3923 Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Sat, 11 Dec 2021 16:26:07 -0700
Subject: [PATCH] Update setup.py

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index ee163bc79f46a..26b85a41f76b7 100755
--- a/setup.py
+++ b/setup.py
@@ -47,7 +47,7 @@
     "jinja2==3.0.3",
     "PyJWT==2.1.0",
     # PyJWT has loose dependency. We want the latest one.
-    "cryptography==35.0.0",
+    "cryptography==3.4.8",
     "pip>=8.0.3,<20.3",
     "python-slugify==4.0.1",
     "pyyaml==6.0",
