From 5ee650bd87eaad7e561c62e6f226395773e289ac Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Wed, 2 Mar 2022 18:03:26 -0700
Subject: [PATCH] Update setup.cfg

---
 setup.cfg | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/setup.cfg b/setup.cfg
index 24617a63..0aeb9f9d 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -37,14 +37,14 @@ install_requires =
 	attrs==21.2.0
 	atomicwrites==1.4.0
 	awesomeversion==22.2.0
-	bcrypt==3.1.7
+	bcrypt==3.2.0
 	certifi>=2021.5.30
 	ciso8601==2.2.0
 	httpx==0.21.3
 	ifaddr==0.1.7
 	jinja2==3.0.3
 	PyJWT==2.1.0
-	cryptography==35.0.0
+	cryptography==3.4.8
 	pip>=21.0,<22.1
 	python-slugify==4.0.1
 	pyyaml==6.0
