From 0aa28a2dc03cafb585caa2e932677937f6eb88db Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Wed, 2 Feb 2022 17:11:01 -0700
Subject: [PATCH] Update setup.cfg

---
 setup.cfg | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.cfg b/setup.cfg
index 970a736b68f1e..1de41f104c4d3 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -47,7 +47,7 @@ install_requires =
     jinja2==3.0.3
     PyJWT==2.1.0
     # PyJWT has loose dependency. We want the latest one.
-    cryptography==35.0.0
+    cryptography==3.4.8
     pip>=8.0.3,<20.3
     python-slugify==4.0.1
     pyyaml==6.0
