From 6bc4f6353847a962d3623ef132cf56ef2bfb18c1 Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Fri, 8 Apr 2022 09:35:29 -0600
Subject: [PATCH] Update setup.cfg

---
 setup.cfg | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/setup.cfg b/setup.cfg
index 704b9052..494ade4c 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -11,7 +11,7 @@ install_requires =
 	attrs==21.2.0
 	atomicwrites==1.4.0
 	awesomeversion==22.5.2
-	bcrypt==3.1.7
+	bcrypt==3.2.0
 	certifi>=2021.5.30
 	ciso8601==2.2.0
  # When bumping httpx, please check the version pins of
